src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
+    private static final int REPORT_PARTS = 3; // {"supply", "buy", "result"}
+    private static final String CSV_SEPARATOR = ",";
+    private static final int SUPPLY_REPORT_INDEX = 0;
+    private static final int BUY_REPORT_INDEX = 1;
+    private static final int RESULT_REPORT_INDEX = 2;
+    private static final int REPORT_PART_IN_LINE_INDEX = 0;
+    private static final int RESULT_IN_LINE_INDEX = 1;
+    private static final String REPORT_FORMAT = "supply,%d%nbuy,%d%nresult,%d";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = readAndProcessFile(fromFileName);
```

Comment:
```suggestion
        String fileContent= readFile(fromFileName);
        String report = generateReport(fileContent);
```
If you have "And" in the method name it means you can split it.
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/d4931641-4f61-421a-bf99-56ad6caa4fe5)